rd_ui/app/views/visualizations/markercluster.html

```diff
@@ -0,0 +1,12 @@
+<div class="markercluster-viz markercluster-container">
+
+  <leaflet
+    lf-center="center"
+    markers="markers"
+    layers="layers"
+    defaults="visualization.options.leaflet"
+    width={{visualization.options.general.width+'%'}}
```

Comment:
I think it will be better to allow it take 100% of the container element, as usually no one would like to leave empty space and there is no point in making it bigger than the container.